File: app/src/main/java/com/example/testingapplication/datamodel/AbsoluteLayout.java
package com.example.testingapplication.datamodel;

import com.fasterxml.jackson.annotation.JsonProperty;
import java.util.ArrayList;

public class AbsoluteLayout{
    @JsonProperty("ImageView")
    public ArrayList<ImageView> imageView;
    @JsonProperty("TextView")
    public ArrayList<TextView> textView;
    @JsonProperty("_xmlns:android")
    public String xmlnsAndroid;
    @JsonProperty("_xmlns:app")
    public String xmlnsApp;
    @JsonProperty("_android:layout_width")
    public String androidLayoutWidth;
    @JsonProperty("_android:layout_height")
    public String androidLayoutHeight;
    @JsonProperty("_android:background")
    public String androidBackground;
}

File: app/src/main/java/com/example/testingapplication/datamodel/AbsoluteLayout.kt
package com.example.testingapplication.datamodel

import com.fasterxml.jackson.annotation.JsonProperty
import java.util.ArrayList

class AbsoluteLayout {
    @JsonProperty("ImageView")
    var imageView: ArrayList<ImageView>? = null

    @JsonProperty("TextView")
    var textView: ArrayList<TextView>? = null

    @JsonProperty("_xmlns:android")
    var xmlnsAndroid: String? = null

    @JsonProperty("_xmlns:app")
    var xmlnsApp: String? = null

    @JsonProperty("_android:layout_width")
    var androidLayoutWidth: String? = null

    @JsonProperty("_android:layout_height")
    var androidLayoutHeight: String? = null

    @JsonProperty("_android:background")
    var androidBackground: String? = null
}